### PR TITLE
Backend dev writing

### PR DESCRIPTION
**Pull request by selim-karaduman**
Here is the implementation for suggestion of writings.

### CODE REVIEW

File: backend/src/main/java/com/example/backend/model/writing/Suggestion.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "id_generator")
+    @SequenceGenerator(name="id_generator", sequenceName = "suggestion_id_seq", allocationSize = 1)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "language_id")
+    private Integer languageId;
+
```
Comment (yavuzDemir21) [inline backend/src/main/java/com/example/backend/model/writing/Suggestion.java:18]: How will they push language id? Will the page send another request to fetch all languages from language endpoint?

**Review by yavuzDemir21 — APPROVED** [review]
Apart from the question I asked, pr seems totally fine!

File: backend/src/main/java/com/example/backend/model/writing/Suggestion.java (hunk 1 above)
Comment (selim-karaduman) [reply backend/src/main/java/com/example/backend/model/writing/Suggestion.java:18]: They can use /lang endpoint to get all ids.

Comment (yavuzDemir21) [reply backend/src/main/java/com/example/backend/model/writing/Suggestion.java:18]: Okay nice.

**Pull request merged**